plugin/src/main/java/io/jenkins/plugins/casc/Attribute.java

```diff
@@ -249,6 +268,51 @@ public CNode describe(Owner instance, ConfigurationContext context) throws Confi
         }
     }
 
+
+    /**
+     * This function is for the JSONSchemaGeneration
+     * @param instance
+     * @param context
+     * @return
+     */
+    public CNode describeStructure(Owner instance, ConfigurationContext context) {
+
+        final Configurator c = context.lookup(type);
+        if (c == null) {
+            return new Scalar("FAILED TO EXPORT\n" + instance.getClass().getName()+"#"+name +
+                ": No configurator found for type " + type);
+        }
+        try {
+            Object o = new Object();
+            if(isJsonSchema) {
+                o = getType();
+                if (o == null) {
+                    return null;
+                }
+            }
+
+            // In Export we sensitive only those values which do not get rendered as secrets
+            boolean shouldBeMasked = isSecret(instance);
+            if (multiple) {
+                Sequence seq = new Sequence();
+                if (o.getClass().isArray()) o = Arrays.asList((Object[]) o);
+                if(o instanceof Iterable) {
+                    for (Object value : (Iterable) o) {
+                        seq.add(_describe(c, context, value, shouldBeMasked));
+                    }
+                }
+                return seq;
+            }
+            return _describe(c, context, o, shouldBeMasked);
+        } catch (Exception | /* Jenkins.getDescriptorOrDie */AssertionError e) {
```

Comment:
`/* Jenkins.getDescriptorOrDie */AssertionError` is it a real case? It might happen in the test code, but I am not sure why test specifics gets leaked to the production codebase

Reply:
I am not totally sure about this, could remove it though